test/unit/recursive-delete.test.ts

```diff
@@ -13,6 +14,7 @@ describe('recursiveDelete', () => {
     expect.assertions(1)
     try {
       await recursiveCopy(resolveDataDir, testResolveDataDir)
+      await fs.symlink('./aa', join(testResolveDataDir, 'symlink'))
```

Comment:
Does `aa` exist? I thought the first parameter is the existing file (the target)?

Reply:
Yeah `aa` exists, seems target is first for `symlink` https://nodejs.org/api/fs.html#fspromisessymlinktarget-path-type